src/components/Navbar.js

```diff
@@ -1,123 +1,135 @@
-import React from "react";
-import { Link } from "gatsby";
+import React, { useState } from "react";
+import { Link, useStaticQuery, graphql } from "gatsby";
+import Img from "gatsby-image";
 import headerBackground from "../img/header-bg.png";
-import logoImg from "../img/header-rosie.png";
-import logoImgMobile from "../img/header-mobile.png";
-import maskNowImg from "../img/header-masksnow.png";
 
 const HamburgerLine = () => {
   return <span />;
 };
 
-const Navbar = class extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      active: false,
-      navBarActiveClass: ""
-    };
-  }
+const Navbar = () => {
+  const data = useStaticQuery(graphql`
+    query {
+      rosie: file(relativePath: { eq: "header-rosie.png" }) {
+        childImageSharp {
+          fixed(height: 300) {
+            ...GatsbyImageSharpFixed
+          }
+        }
+      }
+      mobile: file(relativePath: { eq: "header-mobile.png" }) {
+        childImageSharp {
+          fixed(height: 300) {
+            ...GatsbyImageSharpFixed
```

Comment:
```suggestion
      rosie: file(relativePath: { eq: "header-rosie.png" }) {
        childImageSharp {
          fixed(height: 300) {
            ...GatsbyImageSharpFixed_withWebp_tracedSVG
          }
        }
      }
      mobile: file(relativePath: { eq: "header-mobile.png" }) {
        childImageSharp {
          fixed(height: 300) {
            ...GatsbyImageSharpFixed_withWebp_tracedSVG
```

What do you think about `GatsbyImageSharpFixed_withWebp_tracedSVG` fragment? I usually use this one because of: 

1) webp extension is smaller
2) The traced SVG makes the screen doesn't flick when the image is loading

Reply:
Sounds great! We can move forward with those two optimized images with webp for now and come back to the banner logo.